src/SessionInstance.php

```diff
@@ -188,13 +198,17 @@ public function destroy()
     {
         $this->init();
 
-        # Start the session up, but ignore the error about headers already being sent
-        @session_start();
```

Comment:
It is, but then `session_write_close()` is called meaning emptying `$_SESSION` has no effect, we must restart the session to empty it